src/components/i-personal-data.ts

```diff
@@ -0,0 +1,10 @@
+export interface IPersonalDataForm {
+  companyName: string;
+  city: string;
+  street: string;
+  postcode: string;
+  nip: string;
+  phoneNumber: number | null;
+  email: string,
+  bankAccount: string;
+}
```

Comment:
Missing empty line at the end:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
lint adjusted